### ISSUE TITLE
Statistics command in cogs.commands.py to large

### ISSUE THREAD

**Issue by JakeLion** [DistinctNoot/Nootbot#4]
The command uses inline for all the elements, causing it to take up a large amount of space. Maybe some of the elements could be not be inline to make the embed more horizontal? Might help to keep the chat less crowded and make the embed look better.

**Comment by beeeeez0** [comment]
Maybe it will, I will review the code again.

**Comment by JakeLion** [comment]
Yeah, you can just remove `inline=False` on most of the lines or all of them, discord will automatically move fields and embed elements.
